src/Pack/Policy/Policy.csproj

```
@@ -1,60 +1,56 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <PackageId>Azure.ApiManagement.PolicyToolkit</PackageId>
-    <PackageVersion>0.0.1</PackageVersion>
-    <Authors>Mielek</Authors>
-<!--    <PackageLicenseUrl>http://LICENSE_URL_HERE_OR_DELETE_THIS_LINE</PackageLicenseUrl>-->
-<!--    <PackageProjectUrl>http://PROJECT_URL_HERE_OR_DELETE_THIS_LINE</PackageProjectUrl>-->
-<!--    <PackageIconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</PackageIconUrl>-->
-<!--    <RepositoryUrl>http://REPOSITORY_URL_HERE_OR_DELETE_THIS_LINE</RepositoryUrl>-->
-<!--    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>-->
-<!--    <Description>Azure.ApiManagement.PolicyToolkit</Description>-->
-<!--    <PackageReleaseNotes>Summary of changes made in this release of the package.</PackageReleaseNotes>-->
-<!--    <Copyright>Copyright</Copyright>-->
-<!--    <PackageTags>Azure.ApiManagement.PolicyToolkit</PackageTags>-->
-    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <PropertyGroup>
+        <PackageId>Azure.ApiManagement.PolicyToolkit</PackageId>
```

I'd add something at the end, given testing is doing something similar
```suggestion
        <PackageId>Azure.ApiManagement.PolicyToolkit.Policies</PackageId>
```

---

I would go to with the Authoring suffix as this library is for Authoring policies. How do you like that?
```suggestion
        <PackageId>Azure.ApiManagement.PolicyToolkit.Authoring</PackageId>
```

---

Fine by me

---

Let's reflect this in namespaces as well though

---

Ok, let's address this in next PR. It looks like packaging and packages needs to be revamped.